rpcs3/Emu/RSX/gcm_enums.h

```diff
@@ -992,8 +992,35 @@ enum
 
 enum Method
 {
+	/*
```

Comment:
Except the old names were inaccurate too! Jumping and calling isn't determined by flags! It's determined like ARM instructions, so lots of masking and checking.